### PR TITLE
Dev-3364 Crypto Miner > Add instruction for proper mining in AMD cards

### PR DESCRIPTION
**Pull request by jonelrempis**
## Ticket references
- [Dev-3364](https://freedom.myjetbrains.com/youtrack/issue/Dev-3364)

## Summary of changes
- Added translations for Setup instructions

## Checklist
- [ ] bugfix
- [x] new feature
- [ ] breaking change
- [ ] added unit tests
- [ ] changes introduced has been discussed and approved
- [ ] requires manual testing
- [ ] documentation has been updated

## Testing
Test instructions are [here](https://gitlab.com/mcnfreedom/vn-team/crypto/crypto.miner.client/merge_requests/282#b9bc2a5050f17be648233a9957bf797f4aa17f31)

### CODE REVIEW

File: en/index.json [hunk 1]
```diff
@@ -316,5 +316,12 @@
     "level_up": "Level Up",
     "earn_support_points": "Earn support points",
     "current_sp": "Current SP"
+  },
+  "setup_instruction": {
+    "title": "Setup Instructions",
+    "desc": "How to manually setup GPU cards. Ex. AMD Cards",
+    "amd_cards": {
```
Comment (pandafox) [inline en/index.json]: Should we just use the exact vendor key? "amd"

Comment (jonelrempis) [reply en/index.json]: Ah. Sure.

**Review by pandafox — APPROVED** [review]
Reminder: We'd also need to change the tab-key

**Review by Jhoem — APPROVED** [review]

**Review by zsannenmariano — APPROVED** [review]

**Review by cedricabuso — APPROVED** [review]

**Comment by cedricabuso** [comment]
image

**Comment by ferdrian-melarpis** [comment]
image

**Pull request merged**